.cli/init.js

```diff
@@ -51,11 +67,49 @@ const validateEmulsifyConfig = (config) => {
   }
 };
 
+/**
+ * Takes an array of objects describing the origin and destination of a given file,
+ * then moves each specified file according to it's to/from properties.
+ *
+ * @param {Array<{ to: string, from: string }>} files array of objects depicting the origin and destination of a given file.
+ * @returns void.
+ */
 const renameFiles = (files) =>
   files.map(({ from, to }) =>
     fs.renameSync(path.join(__dirname, from), path.join(__dirname, to)),
   );
 
+/**
+ * Takes a machineName, and returns a fn that, when called with a str,
+ * replaces all instances of `emulsify` with the given machineName.
+ *
+ * @param {string} machineName string that should replace emulsify.
+ * @returns {function} fn that when called with a str, replaces all instances of `emulsify` with the given machineName.
+ */
+const strReplaceEmulsify = (machineName) => (str) =>
+  str.replace(/emulsify/g, machineName);
```

Comment:
@patrickocoffeyo Is this case sensitive?  Do we need to also replace "Emulsify"?

Reply:
Maybe this is the only instance we have to worry about and this is handled in other code you wrote already?  https://github.com/emulsify-ds/emulsify-drupal/blob/master/emulsify.info.yml#L1

Reply:
@ccjjmartin correct, that property is handled separately. Rather than replacing all instances of `emulsify` or `Emulsify` within a file, I am targeting specific properties.

Reply:
:+1: